lab-mar-21/gulp/js/compute.js

```diff
@@ -88,14 +94,36 @@ exports.multiply = multiply;
 
 //problemThree sumAndMultiply
 function sumAndMultiply(aa, bb, cc) {
-  console.log(aa + " and " + " bb " + " cc " + "sum to " + sum(aa, bb, cc));
-  console.log("the numbers " + aa + " and " + " bb " + " cc " + "have a product of " + multiply(aa, bb, cc));
+  console.log(aa + " and " + bb + cc + "sum to " + sum(aa, bb, cc));
+  console.log("the numbers " + aa + " and " + bb + cc + "have a product of " + multiply(aa, bb, cc));
   return [sum(aa, bb, cc), multiply(aa, bb, cc)];
 }
 
 exports.sumAndMultiply = sumAndMultiply;
 
 //problemFour
+function sumArray(arr) {
+  var result = 0;
+  for (var i = 0; i < arr.length; i++) {
+    result += arr[i];
+    console.log(arr[i]);
+  }
+  return result;
+}
+sumArray([1, 2, 3]);
+exports.sumArray = sumArray;
+
+//problemFiveFive
+function multiplyArray(arrMult) {
+  var result = 1;
+  for (var i = 0; i < arrMult.length; i++) {
+    result = result * arrMult[i];
+    console.log(arrMult[i]);
+  }
+  return result;
+}
+
+exports.multiplyArray = multiplyArray;
 
```

Comment:
nice. I like that you used console.log() a bunch more than I did... I will try to remember that :)